src/App.js

```diff
@@ -1,9 +1,10 @@
+import Calculator from './components/Calculator';
 import './styles/main.css';
 
 function App() {
   return (
     <div className="App">
-      <h1>Math magician project setup</h1>
+      <Calculator />
```

Comment:
According to the requirements:

![image](https://user-images.githubusercontent.com/57421823/169986905-ef501b72-b1e5-46b6-a22a-c847690ee07f.png)

Therefore:

- [ ] Please, refactor this component to be class-based.  Kindly remember that the `App` component, is the main one.